keras_cv/models/backbones/vit_det/vit_det_backbone.py

```diff
@@ -123,6 +124,11 @@ def __init__(
             # Use common rescaling strategy across keras_cv
             x = keras.layers.Rescaling(1.0 / 255.0)(x)
 
+        # VITDet scales inputs based on the standard ImageNet mean/stddev.
```

Comment:
So we want to apply both types of rescaling? I'm a bit confused.

Reply:
The `include_rescaling` check and associated rescaling layer make sure that the inputs are scaled from 0-1.

The subsequent bit rescales that using the mean and stddev of ImageNet, with the prior assumption that inputs are scaled 0-1